cartridges/int_algolia/steptypes.json

```
@@ -314,6 +314,78 @@
                         }
                     ]
                 }
+            },
+            {
+                "@type-id": "custom.algoliaContentIndex",
+                "@supports-parallel-execution": true,
+                "@supports-site-context": true,
+                "@supports-organization-context": false,
+                "description": "Index all contents. The list of indexed attributes is configurable.",
+                "module": "int_algolia/cartridge/scripts/algolia/steps/algoliaContentIndex.js",
+                "read-function": "read",
+                "process-function": "process",
+                "write-function": "send",
+                "total-count-function": "getTotalCount",
+                "before-step-function": "beforeStep",
+                "before-chunk-function": "",
+                "after-chunk-function": "",
+                "after-step-function": "afterStep",
+                "chunk-size": 500,
+                "transactional": false,
+                "timeout-in-seconds": "7200",
+                "parameters": {
+                    "parameter": [
+                        {
+                            "@name": "attributeList",
+                            "@type": "string",
+                            "description": "A comma-separated list of attributes will be indexed.",
+                            "@required": false,
+                            "@trim": true,
+                            "default-value": "id,name,description,url,body,page"
+                        },
+                        {
+                            "@name": "failureThresholdPercentage",
+                            "@type": "long",
+                            "description": "Allowed percentage of records that fails to be indexed before marking the job in error. Default: 0. When indexingMethod=fullContentReindex, the temporary index is not moved to production if the failure percentage exceeds this threshold.",
+                            "@required": false,
+                            "min-value":"0",
+                            "max-value":"100"
+                        },
+                        {
+                            "@name": "splitterTag",
+                            "@type": "string",
+                            "description": "The tag used to split the content into chunks. If not specified, tag splitment will not work, but only maximum bytes rule per chunk will be used.",
+                            "@required": false,
+                            "@trim": true
+                        },
+                        {
+                            "@name": "indexingMethod",
```

To not make it confusing with the `indexingMethod` parameter of Products jobs, we should rename it.
```suggestion
                            "@name": "includedContent",
```